src/api/public/apidocs/paths/build__workerstatus.yaml

```
@@ -0,0 +1,113 @@
+get:
+  deprecated: true
+  summary: Lists status of workers, jobs, backend services and general statistics.
+  description: |
+    Lists status of workers, running jobs, waiting jobs, status of the backend services and general statistics.
+
+    Other ways to obtain the same information as with this endpoint are:
+      * `GET /worker/_status`
+      * `GET /status/workerstatus`
+      * `GET /build/_workerstatus`
+      * `GET /worker/status`
```

```suggestion
    This endpoint is exactly the same as `GET /worker/_status`, please use that one.
```
For a deprecated endpoint, we should recommend using the endpoint which is not deprecated. No need to also list the already deprecated alternatives.

The same applies for the rest of the deprecated endpoints.

---

Addressed.